OpenRA.Mods.Common/Widgets/Logic/ModBrowserLogic.cs

```diff
@@ -148,10 +155,29 @@ void SelectMod(ModMetadata mod)
 			var selectedIndex = Array.IndexOf(allMods, mod);
 			if (selectedIndex - modOffset > 4)
 				modOffset = selectedIndex - 4;
+
+			loadButton.Text = modInstallStatus[mod] ? "Load Mod" : "Install Assets";
```

Comment:
The screen that pops up after "Install Assets" says "Game assets not found!". This wording assumed that we kept the "Load Mod" button text.  Can you please either keep the button text, or change the title of the dialog?